apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -114,6 +115,13 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
     RouteStops.by_direction(stops[route.id], shapes, route, direction_id)
   end
 
+  @spec get_shuttle_data(Plug.Conn.t()) :: [Phoenix.HTML.safe()]
+  def get_shuttle_data(conn) do
+    now = conn.assigns[:date_time] || Util.now()
```

Comment:
I think we can make this `conn.assigns.date_time` (without the or-default) since we expect it to always be there when the `DateTime` plug is used; it has its own defaulting logic.

Reply:
That's great, done